src/DebugConfigurationProvider.ts

```diff
@@ -31,6 +31,12 @@ export class BrightScriptDebugConfigurationProvider implements DebugConfiguratio
     public async resolveDebugConfiguration(folder: WorkspaceFolder | undefined, config: BrightScriptDebugConfiguration, token?: CancellationToken): Promise<DebugConfiguration> {
         //make sure we have an object
         config = config ? config : {} as any;
+        
+        //Check for depreciated Items
+        if (config.debugRootDir){
```

Comment:
We still need to support debugRootDir for a while, so you should put `debugRootDir` INTO `sourceDirs` in this situation instead of throwing an error. Something like this:

```typescript
if(config.debugRootDir){
    if(config.sourceDirs){
        throw new Error("Cannot set both debugRootDir AND sourceDirs");
    } else{
        config.sourceDirs = [config.debugRootDir];
    }
}
```